src/main/java/com/faforever/api/config/FafApiProperties.java

```diff
@@ -250,4 +251,14 @@
   public static class Tutorial {
     private String thumbnailUrlFormat;
   }
+
+  @Data
+  public static class ForgedAllianceExe {
+    /**
+     * Allowed file extension of uploaded ForgedAlliance.exe.
+     */
+    private Set<String> allowedExtensions = ImmutableSet.of("exe");
```

Comment:
This should be obsolete, why would we allow any other name than the one we expect?

Reply:
I agree with you, just did the same way as it's around in code.